ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/capella/statetransition/epoch/EpochProcessorCapella.java

```
@@ -0,0 +1,76 @@
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.MutableBeaconState;
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.versions.capella.MutableBeaconStateCapella;
+import tech.pegasys.teku.spec.datastructures.state.versions.capella.HistoricalSummary;
+import tech.pegasys.teku.spec.logic.common.helpers.BeaconStateMutators;
+import tech.pegasys.teku.spec.logic.common.statetransition.epoch.status.ValidatorStatusFactory;
+import tech.pegasys.teku.spec.logic.common.util.BeaconStateUtil;
+import tech.pegasys.teku.spec.logic.common.util.ValidatorsUtil;
+import tech.pegasys.teku.spec.logic.versions.altair.helpers.BeaconStateAccessorsAltair;
+import tech.pegasys.teku.spec.logic.versions.altair.helpers.MiscHelpersAltair;
+import tech.pegasys.teku.spec.logic.versions.bellatrix.statetransition.epoch.EpochProcessorBellatrix;
+import tech.pegasys.teku.spec.schemas.SchemaDefinitions;
+import tech.pegasys.teku.spec.schemas.SchemaDefinitionsCapella;
+
+public class EpochProcessorCapella extends EpochProcessorBellatrix {
+  private final SchemaDefinitions schemaDefinitions;
+
+  public EpochProcessorCapella(
+      final SpecConfigCapella specConfig,
+      final MiscHelpersAltair miscHelpers,
+      final BeaconStateAccessorsAltair beaconStateAccessors,
+      final BeaconStateMutators beaconStateMutators,
+      final ValidatorsUtil validatorsUtil,
+      final BeaconStateUtil beaconStateUtil,
+      final ValidatorStatusFactory validatorStatusFactory,
+      final SchemaDefinitions schemaDefinitions) {
+    super(
+        specConfig,
+        miscHelpers,
+        beaconStateAccessors,
+        beaconStateMutators,
+        validatorsUtil,
+        beaconStateUtil,
+        validatorStatusFactory,
+        schemaDefinitions);
+    this.schemaDefinitions = schemaDefinitions;
+  }
+
+  @Override
+  public void processHistoricalRootsUpdate(final MutableBeaconState state) {
+    // no longer used in capella
```

Do we want to throw an error if this gets called? Or is it ok to silently do nothing?

---

specifically silently do nothign as it'll be called...